stable/ambassador/values.yaml

```diff
@@ -0,0 +1,126 @@
+# Default values for ambassador.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+daemonSet: false
+
+ambassador:
+  id: default
+
+namespace:
+  single: false
+  # name: default
+
+# Additional container environment variable
+env:
+  {}
+  # Exposing statistics via StatsD
+  # STATSD_ENABLED: true
+  # STATSD_HOST: statsd-sink
+
+image:
+  repository: quay.io/datawire/ambassador
+  # tag:
+  pullPolicy: IfNotPresent
+  # imagePullSecrets:
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: LoadBalancer
+
+  # Note that target http ports need to match your ambassador configurations service_port
+  # https://www.getambassador.io/reference/modules/#the-ambassador-module
+  http:
+    enabled: true
+    port: 80
+    targetPort: 80
+    # nodePort: 30080
+
+  https:
+    enabled: true
+    port: 443
+    targetPort: 443
+    # nodePort: 30443
+
+  # annotations:
+  #   getambassador.io/config: |
+  #     ---
+  #     apiVersion: ambassador/v1
+  #     kind: Module
+  #     name: ambassador
+  #     config:
+  #       diagnostics:
+  #         enabled: false
+  # loadBalancerSourceRanges:
+  #   - YOUR_IP_RANGE
+
+adminService:
+  create: true
+  type: ClusterIP
+  port: 8877
+  # NodePort used if type is NodePort
+  # nodePort: 38877
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+volumes: {}
+
+volumeMounts: {}
+
+podAnnotations:
+  prometheus.io/scrape: "true"
```

Comment:
These shouldn't be set by default, especially since exporter isn't enabled by default. Also some people use Prometheus Operator and would prefer a ServiceMonitor.